using-mongodb-panache/src/main/resources/META-INF/resources/index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>using-mongodb-panache - 1.0-SNAPSHOT</title>
```

Comment:
We would drop the `using-` from here too.

Reply:
done